src/PointwiseFunctions/Elasticity/PotentialEnergy.hpp

```diff
@@ -0,0 +1,44 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Tags.hpp"
+#include "Elliptic/Systems/Elasticity/Tags.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Tags.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/ConstitutiveRelation.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/Tags.hpp"
+
+namespace Elasticity {
+
+template <size_t Dim>
+Scalar<DataVector> evaluate_potential_energy(
```

Comment:
name this just `potential_energy`, or better `potential_energy_density`. Functions usually compute or evaluate things so no need to prefix with `evaluate_` :)

The other functions and tags that deal with the pot energy density should probably also get the `density` in their names.